packages/types/src/organizationMembership.ts

```diff
@@ -52,6 +100,8 @@ export type OrganizationPermission =
   | 'org:sys_memberships:delete'
   | 'org:sys_domains:read';
 
+export type OrganizationPermission = OrganizationSystemPermission | OrganizationCustomPermission;
```

Comment:
Does this mean consumers that use TS are required to override the global type? Without the use `Autocomplete` it seems like we aren't accepting arbitrary values now.

Reply:
I have reverted these changes and the current implementation continues to use `Autocomplete<>`